### ISSUE TITLE
Forum not reachable (502)

### ISSUE THREAD

**Issue by KristofKoch** [GenericMappingTools/website#151]
The [GMT forum](https://forum.generic-mapping-tools.org) is not reachable for me as of now (03FEB2024 08:51 UTC). I only get a "502 Bad Gateway" response.

**Comment by anbj** [comment]
Same for me, when I checked an hour ago.

**Comment by KristofKoch** [comment]
Still not reachable for me. Does @PaulWessel has some insight?

**Comment by PaulWessel** [comment]
Hoping for help from weiji…

**Comment by KristofKoch** [comment]
Maybe a notice on the GMT website about the forum being down would be helpful to other users?

**Comment by maxrjones** [comment]
Is this discourse server maintained by SOEST ITS or our team? Seems like it could relate to a failed update.

**Comment by PaulWessel** [comment]
Ross tried to install a package weiji wanted but failed. If anyone can help please do. We hope to move it all to Earth scope

**Comment by KristofKoch** [comment]
As all the forum links are pointing to a white page at the moment: Could someone with the proper privileges put up a simple html site saying something like

**Comment by joa-quim** [comment]
Permissions I may have but have no idea where those pages really live.

**Comment by KristofKoch** [comment]
> no idea where those pages really live.

A `traceroute forum.generic-mapping-tools.org` ends inside SOEST (`gmtserver.soest.hawaii.edu`) but I assume you knew that basic bit already.

**Comment by joa-quim** [comment]
Yes, but next come the web servers and all that stuff.

**Comment by leouieda** [comment]
Is Ross able to restore from a back up?

**Comment by PaulWessel** [comment]
Dont know since i am not well omformed. Ross?

**Comment by Esteban82** [comment]
So, do we have to wait for Ross or Weiji to fix it? is there something we could do? Does Ross know about this problem? Should I write to him?

**Comment by PaulWessel** [comment]
He knows

**Comment by PaulWessel** [comment]
I was successful in getting a new install of discourse to come up with a
successful restore of the data.  I am in the midst of rebuilding the whole
gmtserver.  Once that is done, I will bring it up.  Hopefully this week
unless I hit another issue.

ross


/---------------------------------------------------------------\
| Ross J. Ishida                 University of Hawaii, Manoa    |
| Network Administrator          School of Ocean and Earth      |
| Research Computing Facility    Science and Technology(SOEST)  |
| 1680 East West Road            Phone: (808)956-2558           |
| POST #820                      FAX:   (808)956-9256           |
| Honolulu, HI     96822         email: ***@***.*** |
\---------------------------------------------------------------/

**Comment by PaulWessel** [comment]
system is finally up.  please login and make sure everything is working.

ross

/---------------------------------------------------------------\
| Ross J. Ishida                 University of Hawaii, Manoa    |
| Network Administrator          School of Ocean and Earth      |
| Research Computing Facility    Science and Technology(SOEST)  |
| 1680 East West Road            Phone: (808)956-2558           |
| POST #820                      FAX:   (808)956-9256           |
| Honolulu, HI     96822         email: ***@***.*** |
\---------------------------------------------------------------/

**Comment by joa-quim** [comment]
Good news, thanks, but it looks we lost all images in posts.

**Comment by anbj** [comment]
Maybe it'll force the curious to build the images themselves and learning a lot in the process :)

**Comment by seisman** [comment]
> Good news, thanks, but it looks we lost all images in posts.

Similar to this https://meta.discourse.org/t/uploads-recover-from-tombstone-not-resurrecting-images/78295. Perhaps someone must run `rake:recover_from_tombstone` to recover the images?

**Comment by Esteban82** [comment]
Can this be close?

**Comment by Esteban82** [comment]
Yes, the forum is back. I am not sure if all the images were recovered.
